Hindi/hindi.md

```diff
@@ -0,0 +1 @@
+?? ??? ????? ????? ??? ??????? ???? ??? ??? ??? ????? ?? ???? ???? ??? ????? ????? ?? ????? ?????? ???? ?? ??? ?? ??? ??? ????? ??? ?? ???? ??, ?? ?? ????? ????? ??? ??????? ???? ???? ??, ??? ????????? ?? ???? ??? ???? ??? ?? ?? ??? ?? ???? ???
```

Comment:
This file doesn't seem to have uploaded correctly. I'm not sure what the problem is, but you may want to try to fix that.